recipe/meta.yaml

```diff
@@ -45,7 +45,7 @@ requirements:
     - gshhg-gmt  # [not win]
     - dcw-gmt  # [not win]
     - ffmpeg
-    - graphicsmagick
+    # - graphicsmagick
```

Comment:
https://github.com/conda-forge/graphicsmagick-feedstock/pull/11 should fix this and we will be able to un-comment. Not sure if this is optional or not.

Reply:
Yes, graphicsmagick is less commonly used in GMT.

Reply:
Do you want to merge it before we get https://github.com/conda-forge/graphicsmagick-feedstock/pull/11 or do you want to wait for that one?

Reply:
We can merge this PR without graphicsmagick, and add graphicsmagick back after https://github.com/conda-forge/graphicsmagick-feedstock/pull/11 is merged.